ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/activities/ExpenseClaimListActivity.java

```diff
@@ -151,6 +152,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
 					GeolocationViewActivity.class);
 			startActivityForResult(geolocationViewIntent, GeolocationRequestCode.SET_GEOLOCATION);
 			return true;
+		case R.id.display_geolocations_on_map:
+			Intent geolocationMapViewIntent = new Intent(ExpenseClaimListActivity.this,
+					MapViewActivity.class);
+			geolocationMapViewIntent.putExtra("requestCode", GeolocationRequestCode.DISPLAY_GEOLOCATIONS);
```

Comment:
`"requestCode"` should be put into the `IntentExtraIDs` class with the rest of 'em.